### DISCUSSION TITLE
How can I rebind moving around panes?

### DISCUSSION THREAD

**Discussion by lars-vc** [jesseduffield/lazygit#1932] [Q&A]
Moving around panes can be done using the arrow keys, but I would rather use alt-hjkl. How can this be achieved?

**Comment by tg0h** [comment]
have a look at https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#default

In particular, look at prev/nextBlock.

However, you will need to take care that your remap does not clash with existing keymaps.

<img width="953" alt="image" src="https://user-images.githubusercontent.com/7954426/173211910-b6b036ed-00a5-48e8-a59f-5389e8e02d15.png">
